internal/orchestrator/orchestrator.go

```diff
@@ -71,6 +71,7 @@ type Orchestrator interface {
 	GetSubscriptions(ctx context.Context, ns string, filter database.AndFilter) ([]*fftypes.Subscription, error)
 	GetSubscriptionByID(ctx context.Context, ns, id string) (*fftypes.Subscription, error)
 	CreateSubscription(ctx context.Context, ns string, subDef *fftypes.Subscription) (*fftypes.Subscription, error)
```

Comment:
Do we still need this function or should `CreateUpdateSubscription` be the only thing that gets used now?

Reply:
`CreateSubscription` is used on `POST` and `CreateUpdateSubscription` is used on `PUT`.
Within orchestrator and events, they share a common implementation function `createUpdateSubscription` that takes a boolean to distinguish, to avoid code duplication.